plugin/pkg/scheduler/algorithm/predicates/utils.go

```diff
@@ -89,3 +91,24 @@ func GetEquivalencePod(pod *v1.Pod) interface{} {
 type EquivalencePod struct {
 	ControllerRef metav1.OwnerReference
 }
+
+type hostPortInfo struct {
+	protocol string
+	hostIP   string
+	hostPort string
+}
+
+// decode decodes string ("protocol/hostIP/hostPort") to *hostPortInfo object.
+func decode(info string) *hostPortInfo {
```

Comment:
Just nitpick. If it was me, I would define it as something like
```golang
func (h *hostPortInfo) Load(info string) error
```

- It won't be in the namespace of the whole package.
- It will return error when it cannot decode the string.

```golang
h := hostPostInfo{}
err := h.Load(str)
if err != nil {
	return err
}
```

Reply:
I think it is just fine now

Reply:
or at least you call it `decodeHostPortInfo`. anyway I'll leave it to @k82cn to decide.

Reply:
haha I think we can get what we decode from context, not very important to add it to the function name